Samples/PackageWithExternalLocation/cs/PhotoStoreDemo/StartUp.cs

```diff
@@ -169,7 +169,7 @@ private static bool registerSparsePackage(string externalLocation, string sparse
 
                 deploymentOperation.Completed = (depProgress, status) => { opCompletedEvent.Set(); };
```

Comment:
Huh I can't comment on non-changed lines but the way the async is done is bad (have a `ManualResetEvent` bla bla) - Can you switch to just `await`? Line 166 and others following it.

Reply:
Right this is how the package registration code was before I updated it. The initial priority is that code as it's what customers will copy.

Reply:
Style updates are part of https://github.com/microsoft/AppModelSamples/pull/47.